Agent/Measurements/NRMAMeasurementType.m

```diff
@@ -5,35 +5,3 @@
 //  Created by Bryce Buchanan on 9/26/13.
 //  Copyright © 2023 New Relic. All rights reserved.
 //
-
```

Comment:
What was the reason for removing this?

Reply:
I noticed it wasn't being used but it showed up in a crash stack i was looking at. I figured its best to remove the dead dead code if its a safe removal like this.